pupil_src/shared_modules/surface_tracker/surface_tracker_offline.py

```diff
@@ -0,0 +1,781 @@
+"""
+(*)~---------------------------------------------------------------------------
+Pupil - eye tracking platform
+Copyright (C) 2012-2018 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+"""
+
+import os
+import logging
+import time
+import platform
+import multiprocessing
+import csv
+import itertools
+
+logger = logging.getLogger(__name__)
+if platform.system() == "Darwin":
```

Comment:
@papr I am not sure why this has to be set. Can you comment on this?

Reply:
> https://docs.python.org/3.4/library/multiprocessing.html?highlight=process#contexts-and-start-methods

In `main.py` we set the default `start_method` to `spawn` for `Darwin`. IIRC this is to avoid issues with opengl/pylgui in the child processes. Generally, `fork` is more efficient than `spawn` and in the case of surface tracking it is not required to `spawn` child processes. I will ask @mkassner to verify this though.